stock_picking_fillwithstock/views/stock.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form.fillwithstock</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <button name="action_confirm" position="before">
+                <button name="button_fillwithstock" states="draft" string="Fill with stock" type="object" class="oe_highlight" groups="base.group_user" />
```

Comment:
`base.group_user` or `stock.group_stock_user`? There is no guarantee that the first could access to the quants, moves, etc, better to switch to the second I think.